openfisca_core/parameters.py

```diff
@@ -348,10 +367,9 @@ def __init__(self, name = "", directory_path = None, data = None, file_path = No
         else:
             self.file_path = file_path
             _validate_parameter(self, data, data_type = dict, allowed_keys = self._allowed_keys)
-            self.children = {}
-            # We allow to set a reference and a description for a node. It is however not recommanded, as it's only metadata and is not exposed in the legislation explorer.
-            data.pop('reference', None)
-            data.pop('description', None)
+            # We allow to set a reference and a description for a node.
+            self.reference = _item_to_list(data.pop('reference', None))
```

Comment:
Why is it `data.pop()` here when it is `data.get()`in the others ?

Reply:
Because in a parameter node, metadata and children are at the same level (see discussion about a `children` key [here](https://github.com/openfisca/openfisca-core/issues/678).
 
In the line that follows, we treat `data` as only a dict of children parameters, so we need to "pop" the metadata out of the dict so that they are not interpreted as a child.